IssuesFAQ.md

```diff
@@ -0,0 +1,33 @@
+Read before you Post...
```

Comment:
These two lines are just unnecessary.
Obviously if one got to this page, his intent is to read it.